actionpack/lib/abstract_controller/base.rb

```diff
@@ -14,8 +14,16 @@ class ActionNotFound < StandardError
   # expected to provide their own +render+ method, since rendering means
   # different things depending on the context.
   class Base
+    ##
+    # Returns the body of the HTTP response sent by the controller.
     attr_internal :response_body
+
+    ##
+    # Returns the name of the action this controller is processing.
     attr_internal :action_name
+
+    ##
+    # Returns the formats processed by the controller.
```

Comment:
Should this be "formats that can be processed by the controller"? IMHO the sentence is slightly ambiguous right now